FND/Organizations/Organizations.rdf

```diff
@@ -103,5 +101,22 @@
 		<fibo-fnd-utl-av:adaptedFrom>https://www.iso.org/obp/ui/#iso:std:iso-iec:6523:-1:ed-1:v1:en</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:adaptedFrom>https://www.w3.org/TR/vocab-org/#class-organization</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-org-org;OrganizationMember">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&lcc-lr;isMemberOf"/>
+						<owl:allValuesFrom rdf:resource="&fibo-fnd-org-org;Organization"/>
```

Comment:
An instance of this class can be a member of multiple organizations then? Is that deliberate? What properties would be associated with the instance? I really don't follow how this would be used in practice - do we have an example?

Reply:
> An instance of this class can be a member of multiple organizations then? Is that deliberate? What properties would be associated with the instance? I really don't follow how this would be used in practice - do we have an example?

I didn't change that this time around - I think more work is needed here as I said.  I simply moved it from parties to organizations after unwinding the hierarchy.  When we hook it into the pattern properly, I'll create examples.  We do have examples that work for ownership (in FBC, but they exist), but not for organization membership yet.